mobile/src/helpers/fetch.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Functions to Fetch data from the API
+ */
+
+import { Post } from "../types/post.type";
+import { User } from "../types/user.type";
+import { getAuthSession } from "../utils/storage";
+import doRequest from "./request";
+
+export async function searchPosts({
+  search = "",
+  userId = "",
+  collectionId = "",
+  limit = 16,
+  cursorId = "", // cursorId
```

Comment:
What is this comment lol?

Reply:
hmm I should probably be more careful when it comes to copying code from web, also I saw that there is a fetch in yarn workspaces? should this be removed?

Reply:
No, yarn-workspaces is dead. We couldn't figure out how to get it to work. We're just gonna go "worse is better" and keep copy-pasting